### PR TITLE
[16.0][MIG] partner_contact_address_default: Migration to 16.0

### PR DESCRIPTION
**Pull request by Reyes4711-S73**
Standard migration to 16.0

### CODE REVIEW

File: partner_contact_address_default/tests/test_partner_contact_address_default.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Copyright 2020 Tecnativa - Carlos Dauden
+# Copyright 2020 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo.tests import common
+
+
+class TestPartnerContactAddressDefault(common.TransactionCase):
+    def setUp(self):
+        super().setUp()
```
Comment (rousseldenis) [inline partner_contact_address_default/tests/test_partner_contact_address_default.py]: Could you switch to class method ?

**Review by rousseldenis — APPROVED** [review]
LGTM. Minor comment

**Comment by Reyes4711-S73** [comment]
@rousseldenis change done

**Review by BT-apadron — APPROVED** [review]
LGTM

**Comment by Reyes4711-S73** [comment]
@rousseldenis Could you merge ths PR?

**Pull request merged**